data/migrations/20201109234734_seller_profie_and_item.js

```diff
@@ -8,6 +8,13 @@ exports.up = function (knex) {
       tb.string('physical_address', 255);
       tb.text('description');
     })
+    .createTable('buyer_profile', (tb) => {
+      tb.increments();
+      tb.string('buyer_name', 255);
+      tb.string('email_address', 255);
+      tb.string('physical_address', 255);
+      tb.string('description');
```

Comment:
did we want a string here for the avartarURL? I suppose it can wait if we aren't ready to set that up on the frontend, though it wouldn't hurt anything now if it's nullable.

Reply:
Almost forgot about, I'll get on it right away!